suite-native/module-accounts-import/src/components/AccountImportImportedAccount.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+
+import { useNavigation } from '@react-navigation/core';
+
+import { Box, Button, Divider, Text } from '@suite-native/atoms';
+import { AccountListItem } from '@suite-native/accounts';
+import { Account } from '@suite-common/wallet-types';
+import {
+    AccountsImportStackParamList,
+    AccountsImportStackRoutes,
+    AppTabsRoutes,
+    HomeStackRoutes,
+    RootStackParamList,
+    RootStackRoutes,
+    StackToTabCompositeProps,
+} from '@suite-native/navigation';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { AccountImportSummarySection } from './AccountImportSummarySection';
+
+type AccountImportImportedAccountProps = {
+    account: Account;
+};
+
+type NavigationProp = StackToTabCompositeProps<
+    AccountsImportStackParamList,
+    AccountsImportStackRoutes.AccountImport,
+    RootStackParamList
+>;
+
+const contentWrapperStyle = prepareNativeStyle(() => ({
+    marginBottom: 114,
+}));
+
+export const AccountImportImportedAccount = ({ account }: AccountImportImportedAccountProps) => {
+    const { applyStyle } = useNativeStyles();
+    const navigation = useNavigation<NavigationProp>();
+
+    const handleImportAnotherAsset = () =>
+        navigation.navigate(RootStackRoutes.AccountsImport, {
+            screen: AccountsImportStackRoutes.XpubScan,
+            params: {},
+        });
+
+    const handleNavigateToDashboard = () =>
+        navigation.navigate(RootStackRoutes.AppTabs, {
+            screen: AppTabsRoutes.HomeStack,
+            params: {
+                screen: HomeStackRoutes.Home,
+            },
+        });
+
+    return (
+        <AccountImportSummarySection title="Asset already imported">
```

Comment:
I am still confused about naming, should there be `Account already imported?`

Reply:
This was in figma and was approved by copywriter so I just went with it. You can raise a comment if you think it should be renamed. But I guess I wouldn't wait for reply with this PR and just go with this and we can always change it if it's requested.